Flatiron - Tuples chapter.md

```diff
@@ -0,0 +1,115 @@
+
+##What's a Tuple?
+Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
+
+**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
+```
+// Array
+let list = [ "Darth", "Ben", "Luke" ]
+print( list[1] )
+```
```

Comment:
ah, thanks!  Sorry about that.  :)


On Wed, Jun 15, 2016 at 1:23 PM, Jim Campagno <notifications@github.com>
wrote:

> In Flatiron - Tuples chapter.md
> <https://github.com/learn-co-curriculum/swift-curriculum-assignment-tuples-readme/pull/6#discussion_r67206016>
> :
>
> > @@ -0,0 +1,115 @@
> > +
> > +##What's a Tuple?
> > +Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
> > +
> > +**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
> > +```
> > +// Array
> > +let list = [ "Darth", "Ben", "Luke" ]
> > +print( list[1] )
> > +```
>
> When creating code snippets in Swift through Markdown, we like to include
> the word swift in the snippet like so so it formats correctly.
>
> [image: swift snippet]
> <https://camo2.githubusercontent.com/6897b9afc4d355aa04182eed54a33257d65184aa/687474703a2f2f692e696d6775722e636f6d2f623846565a47372e706e67>
> .. turns into this
>
> let name = "Jim"
>
> In applying that logic to the code you wrote above, it would look like
> this instead:
>
> let list = ["Darth", "Ben", "Luke"]
> print(list[1])// prints "Ben"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-co-curriculum/swift-curriculum-assignment-tuples-readme/pull/6/files/3b56b40e6c8a80edd95bd31884418b33cceee139#r67206016>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAhDWZKcoPBn3gK5P1LlCfBjO4s-tnlcks5qMDURgaJpZM4I2iSC>
> .
>